### PR TITLE
#15689 Repro: Dashboard filters should allow multiple selections

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING CI && PENDING REVIEW

### What does this PR accomplish?
- Reproduces #15689 

### How to test this manually?
- `yarn test-cypress-open --spec frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js`
- All tests should pass

### Additional notes:
- The bug was fixed in #15698

### Screenshots:
image

### CODE REVIEW

**Review by daltojohnso — APPROVED** [review]

File: frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js [hunk 1]
```diff
@@ -771,6 +771,71 @@ describe("scenarios > dashboard", () => {
...
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.createDashboard("15689D").then(({ body: { id: DASHBOARD_ID } }) => {
+        // Add filter to the dashboard
+        cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+          parameters: [
+            {
+              name: "Location",
+              slug: "location",
+              id: "5aefc725",
+              type: "string/=",
```
Comment (flamber) [inline frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: Shouldn't that be `type: "location/state"` or is that some semi-test to make sure it works in the future?
`{"name":"Test","description":null,"parameters":[{"name":"State","slug":"state","id":"7342756d","type":"location/state"}]}`

If so, can you also make a test with one of the old styles like Product.Category?
`{"name":"Test","description":null,"parameters":[{"name":"Category","slug":"category","id":"8ac835fa","type":"category"}]}`

**Review by flamber — APPROVED** [review]

File: frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: > Shouldn't that be type: "location/state" or is that some semi-test to make sure it works in the future?

That's what I've got directly from observing network requests.

Comment (flamber) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: Interesting. That's not what I'm getting on release or the CI build. I'm getting the snippets I posted.

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: Just to be sure, I used this exact code BEFORE the fix and could confirm it fails.

Comment (flamber) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: If it fails before and working now, then merge - I'm just getting tired of these regressions, so I just wanted to test the :turd: of out it, including the old styles to ensure upgrades work.

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: This is from a docker image 0.39.0:


And this is from a local dev on `release-x.39.x`:
https://p58.f1.n0.cdn.getcloudapp.com/items/llu5kg6X/c4f2aadf-64e2-41b0-8d48-d5b28b458a5e.gif?source=viewer&v=d59b575d6dbd1431dd3752eed51fe011

Hence the difference.

Comment (flamber) [reply frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js:788]: Exactly. That's 40 style. I don't care anymore - merge it.

**Pull request merged**